pal/input/native/gamepad-input.ts

```diff
@@ -1,7 +1,356 @@
+                negativeButton = Button.RIGHT_STICK_LEFT;
+                positiveButton = Button.RIGHT_STICK_RIGHT;
+                axisValue = this._axisToButtons(value);
+                break;
+            case 6:
+                negativeButton = Button.RIGHT_STICK_DOWN;
+                positiveButton = Button.RIGHT_STICK_UP;
+                axisValue = this._axisToButtons(value);
+                break;
+            default:
+                if (code === 7) { this._nativeButtonState[Button.BUTTON_L2] = value; }
+                else if (code === 8) { this._nativeButtonState[Button.BUTTON_R2] = value; }
+                break;
+            }
+            if (negativeButton && positiveButton && axisValue) {
+                this._nativeButtonState[negativeButton] = axisValue.negative;
+                this._nativeButtonState[positiveButton] = axisValue.positive;
+            }
+        }
+    }
+
+    private _initInputSource () {
+        this._buttonNorth = new InputSourceButton();
+        this._buttonNorth.getValue = () => this._nativeButtonState[Button.BUTTON_NORTH];
+        this._buttonEast = new InputSourceButton();
+        this._buttonEast.getValue = () => this._nativeButtonState[Button.BUTTON_EAST];
+        this._buttonWest = new InputSourceButton();
+        this._buttonWest.getValue = () => this._nativeButtonState[Button.BUTTON_WEST];
+        this._buttonSouth = new InputSourceButton();
+        this._buttonSouth.getValue = () => this._nativeButtonState[Button.BUTTON_SOUTH];
+
+        this._buttonL1 = new InputSourceButton();
+        this._buttonL1.getValue = () => this._nativeButtonState[Button.BUTTON_L1];
+        this._buttonL2 = new InputSourceButton();
+        this._buttonL2.getValue = () => this._nativeButtonState[Button.BUTTON_L2];
+        this._buttonL3 = new InputSourceButton();
+        this._buttonL3.getValue = () => this._nativeButtonState[Button.BUTTON_L3];
+        this._buttonR1 = new InputSourceButton();
+        this._buttonR1.getValue = () => this._nativeButtonState[Button.BUTTON_R1];
+        this._buttonR2 = new InputSourceButton();
+        this._buttonR2.getValue = () => this._nativeButtonState[Button.BUTTON_R2];
+        this._buttonR3 = new InputSourceButton();
+        this._buttonR3.getValue = () => this._nativeButtonState[Button.BUTTON_R3];
+
+        // this._buttonTouchPad = new InputSourceButton();
+        // this._buttonTouchPad.getValue = () => 0;  // TODO: NX unavailable
+        // this._buttonHome = new InputSourceButton();
+        // this._buttonHome.getValue = () => 0;  // TODO: NX unavailable
+
+        this._buttonShare = new InputSourceButton();
+        this._buttonShare.getValue = () => this._nativeButtonState[Button.NS_MINUS];  // TODO: NX only for now
+        this._buttonOptions = new InputSourceButton();
+        this._buttonOptions.getValue = () => this._nativeButtonState[Button.NS_PLUS] || this._nativeButtonState[Button.ROKID_MENU];  // TODO: NX, Rokid only for now
```

Comment:
```suggestion
        this._buttonOptions.getValue = () => this._nativeButtonState[Button.NS_PLUS] || this._nativeButtonState[Button.ROKID_MENU]; 
```

need to remove this todo